projects/storefrontlib/shared/components/card/card.component.ts

```diff
@@ -72,7 +75,8 @@ export class CardComponent implements OnInit {
     this.cancelCard.emit(5);
   }
 
-  delete(): void {
+  delete(event?: Event): void {
+    event?.preventDefault();
```

Comment:
Now that we are using buttons, what happens if we drop the `event` parameter?

Reply:
Still works!